addon/templates/components/polaris-data-table/total.hbs

```diff
@@ -1,22 +1,12 @@
-{{#if (eq total "")}}
-  {{#if (eq index 0)}}
-    {{polaris-data-table/cell
-      fixed=true
-      total=true
-      height=height
-      text=totalsRowHeading
-      truncate=truncate
-    }}
-  {{else if (eq index 1)}}
-    {{polaris-data-table/cell presentational=true height=height}}
-  {{else}}
-    {{polaris-data-table/cell total=true contentType="numeric" height=height}}
-  {{/if}}
-{{else}}
-  {{polaris-data-table/cell
-    total=true
-    height=height
-    contentType="numeric"
-    text=total
-  }}
-{{/if}}
+{{polaris-data-table/cell
+  total=true
+  truncate=truncate
+  height=heights.firstObject
```

Comment:
Should this be `heights[1]` instead of `heights[0]`?

(We have a helper that can allow you to do this as `(get-array-item heights 1)`)

Reply:
Eek good catch